File: Semestrul 4/SDI/lab09-10/server .net/TriathlonModel/Event.cs
namespace Triathlon.Model
{
    public class Event : TEntity<long>
    {
        private string name;
        private string date;
        private string location;
        private string description;
        public Event() : base()
        {

        }
        public Event(long id, string name, string date, string location, string description) : base(id)
        {
            this.name = name;
            this.date = date;
            this.location = location;
            this.description = description;
        }

        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        public string Date
        {
            get { return date; }
            set { date = value; }
        }

        public string Location
        {
            get { return location; }
            set { location = value; }
        }

        public string Description
        {
            get { return description; }
            set { description = value; }
        }
    }
}

File: Semestrul 4/SDI/lab07/java/TriathlonModel/src/main/java/triathlon/model/Event.java
package triathlon.model;


public class Event extends Entity<Long> {
    private String name;
    private String date;
    private String location;
    private String description;

    public Event(Long aLong, String name, String date, String location, String description) {
        super(aLong);
        this.name = name;
        this.date = date;
        this.location = location;
        this.description = description;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}
